.github/workflows/python-package-conda.yml

```diff
@@ -0,0 +1,44 @@
+name: Testing on Ubuntu latest with py 3.8
+
+on:
+  push:
+    branches-ignore:
+      - master
+      - devel
+  pull_request:
+    branches-ignore:
+      - master
+      - devel
+jobs:
+  build-linux:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 5
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Add conda to system path
+      run: |
+        # $CONDA is an environment variable pointing to the root of the miniconda directory
+        echo $CONDA/bin >> $GITHUB_PATH
+    - name: Install dependencies
+      run: |
+        conda env update --file requirements.txt --name base
+    - name: Lint with flake8
+      run: |
+        conda install flake8
+        # stop the build if there are Python syntax errors or undefined names
+        # too many F82 errors, should uncomment the following line
+        # flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+        flake8 . --count --select=E9,F63,F7 --show-source --statistics
+        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+    - name: install tofu
+      run: |
+        pip install -e ".[dev]"
```

Comment:
This file is called conda but the tofu installation is done with pip, I'm not sure I get the point ?

Reply:
pip ? conda ? 
If you want to install from a conda package you first need to create the pack:
 conda build conda_recipe

And then install from that package
conda install -c conda-bld/<name of the created package>

Reply:
mmm you are right, I could build the library with conda, however our `conda-recipe` includes the deployment (we have a `conda-upload.sh` file). I decided to build it with `pip` for simplicity. I am going to rename the file and not use `conda`, would you be ok with that solution ? I still have to write a GA file for conda deployment.

Reply:
OK for that solution